### ISSUE TITLE
Documentation by versions

### ISSUE THREAD

**Issue by yasmanmesa** [smaugho/declex#82]
It's important to make a documentation for each version of DecleX. As is well known, everything that is beginning is constantly changing, with the aim of eliminating deficiencies and adding new facilities. There are many frameworks where documentation is done for each version, so the developer can easily guide and find solutions much faster. Although there are few changes, it is always good to specify them and make them known.

**Comment by smaugho** [comment]
Agree, but in the current state of the project, it would be enough with a page with a migration guide between versions. So in the Wiki can be created this page. All the changes from version 1.0 has been registered in the Release notes as well. 

@yasmanmesa please feel free to contribute with a migration page for the wiki. Thanks 👍
